src/Traits/FieldTypeTrait.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Opstalent\CrudBundle\Traits;
+
+use Symfony\Component\Form\FormTypeInterface;
+
+/**
+ * Trait FieldTypeTrait
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+trait FieldTypeTrait
+{
+    /**
+     * @param string $type
+     * @return bool
+     */
+    protected function isTypeAvailable(string $type): bool
+    {
+        if (is_subclass_of(new $type(), FormTypeInterface::class)) {
```

Comment:
is this works for non-existing classes? maybe something for security will be worth

more clean imho: return is_subclass_of(new $type(), FormTypeInterface::class);